experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -139,6 +139,15 @@ export class SharedTree
 	public readonly view: SharedTreeView;
 	public readonly storedSchema: SchemaEditor<InMemoryStoredSchemaRepository>;
 
+	/**
+	 * Creating multiple editable tree contexts with a single AnchorSet does not work due to have TreeNode caching works.
+	 * This flag is used to detect and error on the easy ways to accidentally do this.
```

Comment:
nit: i don't understand what this is saying

Reply:
should be more clear now.